metafix/src/test/resources/org/metafacture/metafix/integration/lookup/fromJson/toJson/lookupRdfPropertyToSubject/test.fix

```diff
@@ -1 +1 @@
-lookup_rdf("a", "./hcrt.ttl", target: "skos:Concept")
```

Comment:
@blackwinter : shouldn't a file be refrenced with dot-notation from the fix-file?

Reply:
Sorry, I don't follow. As I see it, @dr0i move the file and updated the reference. What are you asking?

Reply:
yes, I moved it to one place to be consumable by all the tests => avoiding multiple uploads of the same 6kB hcrt.ttl .

Reply:
Ah, i didn't see this. Thanks!